### PR TITLE
show if is's a pic or vid

### PR DESCRIPTION
**Pull request by jakobroehrl**
Signed-off-by: Jakob Röhrl <jakob.roehrl@web.de>
solves #417 

grafik

should work with video previews, too.

### CODE REVIEW

**Comment by jakobroehrl** [comment]
@skjnldsv 
1. How could I solve this better then with this ActionButton? I only want to show an video/picure icon
2. How to show this icon over the picture? Currently the picture covers it.
Thanks for you help

**Comment by skjnldsv** [comment]
Hey!
Since there is no action on the button, it would be easier with a simple div, absolutely positioned, with the icon-video class :)

**Comment by jakobroehrl** [comment]
This is working, but it's covered if there are any previews:
grafik

File: src/components/File.vue [hunk 1]
```diff
@@ -132,6 +133,12 @@ export default {
 <style lang="scss" scoped>
 @import '../mixins/FileFolder.scss';
 
+.icon-video {
+	position: absolute;
+	top: 10px;
+	right: 10px;
```
Comment (skjnldsv) [inline src/components/File.vue:139]: Add a `z-index: 5` Or higher if 5 is not enough :)

**Comment by jakobroehrl** [comment]
It's working nice, thanks.

**Review by jancborchardt — APPROVED** [review]

**Pull request merged**